ansible_bender/cli.py

```diff
@@ -70,6 +70,7 @@ def __init__(self):
         debug = False
         if self.args.debug:
             debug = True
+            run_cmd(["export", "ANSIBLE_STDOUT_CALLBACK=debug"])
```

Comment:
this won't work since the variable would only be set in the forked process, not in the ansible-bender process